stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -449,7 +449,19 @@ public void start() throws BundleException {
         System.getProperty("felix.log.level", String.valueOf(Logger.LOG_WARNING)));
     configMap.put(
         FelixConstants.FRAMEWORK_SYSTEMPACKAGES_EXTRA,
-        "sun.misc,sun.nio.ch,com.sun.management,sun.rmi.registry");
+        String.join(
+            ",", "sun.misc,sun.nio.ch", "com.sun.management", "sun.rmi.registry", "org.osgi.*"));
+    configMap.put(
+        FelixConstants.FRAMEWORK_BOOTDELEGATION,
```

Comment:
Seems it s telling to look for these classes in the boot class loader, is this really needed? I mean class loading works in the parent-first, so these will anyway first be looked in the boot.. What's the reason to explicitly specify this?

Reply:
+1 on the reason for this.

Reply:
AFAIK, OSGi does not automatically delegate to the boot class loader (i.e. JRE). When we had dynamic imports it was happening because of that, as far as I understand, but now that we do not use dynamic imports, we have to declare what we want to be available from the JRE.